sled-agent/src/services.rs

```diff
@@ -1038,13 +1046,15 @@ impl ServiceManager {
             .map(|d| zone::Device { name: d.to_string() })
             .collect();
 
-        // Look for the image in the ramdisk first
-        let mut zone_image_paths = vec![Utf8PathBuf::from("/opt/oxide")];
+        let mut zone_image_paths = vec![];
```

Comment:
I swapped the order here because I noticed failures on my local Helios installation, that already had an image in `/opt/oxide`, but which also wanted to run this test (and therefore use an `image_directory_override`).